rsmtool/test_utils.py

```diff
@@ -141,10 +141,10 @@ def check_run_experiment(source,
     check_report(html_report, raise_warnings=False)
 
     # make sure that there are no warnings in the report
-    # but ignore deprecation warnings if appropriate
+    # but ignore warnings if appropriate
     warning_msgs = collect_warning_messages_from_report(html_report)
-    if IGNORE_DEPRECATION_WARNINGS:
-        warning_msgs = [msg for msg in warning_msgs if 'DeprecationWarning' not in msg]
+    if IGNORE_WARNINGS:
+        warning_msgs = [msg for msg in warning_msgs if 'Warning' not in msg]
```

Comment:
Just wondering, won't "Warning" always be in the message? Which messages would not be filtered out here?

Reply:
Same for the repetitions below.

Reply:
Good question! I don't think we need this filtering since previously it was to filter out non-deprecation warnings :)

Reply:
@desilinguist @mulhod I was actually considering not calling `collect_warning_messages_from_report()` when not in STRICT mode but did not want to change it too much. There was some issue with blank spaces, but I've fixed that. What do you guys think?

Reply:
```suggestion
            if not IGNORE_WARNINGS:
                  warning_msgs = collect_warning_messages_from_report(html_report)
                  assert_equal(len(warning_msgs), 0)
```

Reply:
@Frost45 Looks good to me!

Reply:
Please make the change and let the build run.